cg/store/filters/status_pool_filters.py

```diff
@@ -71,13 +71,20 @@ def filter_pools_by_customer(pools: Query, customer: Customer, **kwargs) -> Quer
     return pools.filter(Pool.customer == customer)
 
 
+def filter_pools_by_customers(pools: Query, customers: list[Customer], **kwargs) -> Query:
+    """Return pools by customers."""
+    customer_ids = [customer.id for customer in customers]
+    return pools.filter(Pool.customer_id.in_(customer_ids))
```

Comment:
Not beautiful but apparently there is no support for an "_in" on relationships, meaning that one needs to go from having `Customer` objects to the ids.